ovn-tester/tests/cluster_density.py

```diff
@@ -79,8 +82,9 @@ def run(self, ovn, global_cfg):
                             (j+1)*DENSITY_N_TOT_PODS]
                     ns.add_ports(ports)
                     # add VIPs and backends to cluster load-balancer
-                    ovn.provision_vips_to_load_balancers(
-                            [ports[0:1], ports[2:3], ports[3:4]])
+                    ns.provision_vips_to_load_balancers(
+                            [ports[0:2], ports[2:3], ports[3:4]])
```

Comment:
The first item in this list got changed from `ports[0:1]` to `ports[0:2]`. Was this on purpose?

Reply:
Lorenzo confirmed with me on IRC that this was on purpose.